release-team/role-handbooks/enhancements/README.md

```diff
@@ -50,11 +50,13 @@ Enhancement Freeze through Code Freeze: 4-7 hours a week
 Code Freeze through Release Day: 1-4 hours a week
 
 ## Mentoring Shadows
+
 It is important to select shadows who are interested in learning more about the Kubernetes release process and are able to devote a couple hours a week for attending the Release meeting as well as helping with weekly tasks.  Remember one of these shadows should eventually be taking on the Enhancement Lead role so it is important to think about how you can delegate tasks to them to train them for succession.
```

Comment:
_nit_

We can break down the sentences like this:

The selected shadows should essentially be:
1. Interested in learning more about the Kubernetes release process
2. Able to devote couple of hours every week for attending the Release meeting as well as help with weekly tasks

The shadows should be selected keeping in mind that one of them would eventually be taking up on the Enhancements Lead role. So it is important to train them for succession by delegating tasks.

Reply:
There's an implicit implication that shadows could/should succeed a role lead, but it's good to explicitly call out here. Agreed with the flow change @palnabarun mentions though.

This is a generally useful passage and I wonder if it should also be stated in the Release Team onboarding doc.

Reply:
It could be made more explicit either in onboarding or maybe shadows? In shadows it's called out a bit specifically in the _After Shadowing: What's Next?_ section, but maybe make it clear in the intro at the beginning?

Reply:
I guess it suffices to have it in the _After Shadowing: What's Next?_ section.

As for the lead handbook paragraph, I am thinking if we can write the second sentence as:

The shadows should be selected keeping in mind that one of them may eventually be taking up on the Enhancements Lead role. So it is important to train them for succession by delegating tasks.

> would is replaced by may

Reply:
I've updated and reworded it a tad.

re:the other note about newer references -- I updated them to point towards the ones used for 1.17